packages/renderer/src/lib/ui/ProviderResultPage.svelte

```diff
@@ -150,18 +150,18 @@ function onSeverityClicked(severity: 'critical' | 'high' | 'medium' | 'low' | 's
     <div class="h-full overflow-y-auto w-1/3">
       {#each providers as provider (provider.info.id)}
         <div role="row" class="rounded-lg bg-[var(--pd-content-bg)] mb-4 p-4 flex flex-col">
-          <div class="flex flex-row items-center">
-            <span class="grow">{provider.info.label}</span>
+          <div class="flex flex-row items-center" aria-label="{provider.info.label}">
+            <span class="grow" aria-label="Provider Starting State">{provider.info.label}</span>
```

Comment:
I would not add specific aria-labels into various spans for now. Row + label should be enough.